packages/snowpack/src/commands/dev.ts

```diff
@@ -127,22 +129,41 @@ const sendFile = (
     acceptEncoding = '';
   }
 
-  function onError(err) {
-    if (err) {
-      res.end();
-      console.error(
-        colors.red(`  ✘ An error occurred while compressing ${colors.bold(req.url)}`),
-        err,
-      );
-    }
-  }
-
+  // Handle gzip compression
   if (/\bgzip\b/.test(acceptEncoding) && stream.Readable.from) {
     const bodyStream = stream.Readable.from([body]);
     headers['Content-Encoding'] = 'gzip';
     res.writeHead(200, headers);
-    stream.pipeline(bodyStream, zlib.createGzip(), res, onError);
-    return;
+    stream.pipeline(bodyStream, zlib.createGzip(), res, function onError(err) {
+      if (err) {
+        res.end();
+        console.error(
+          colors.red(`  ✘ An error occurred while compressing ${colors.bold(req.url)}`),
+          err,
+        );
+      }
+    });
+    return
+  }
+
+  // Handle partial requests
+  const {range} = req.headers
```

Comment:
should this go before the gzip handler?

Reply:
I'm not really sure. I will do some reading and see what I come up with.

Reply:
I couldn't find anything saying one way or the other. I went ahead and moved it to before.